docs/sources/gard.md

```diff
@@ -0,0 +1,86 @@
+1. `gard.sssom+`: This is a SSSOM of GARD mappings. For how predicate_id’s were chosen, see “Assumptions” section below.
+Additional columns added: mondo_id, mondo_label, and gard_mondo_mapping_type (which has 3 values: `direct_existing` (
+taken from [mondo_hasdbxref_gard.sssom.tsv](https://raw.githubusercontent.com/monarch-initiative/mondo/master/src/ontology/mappings/mondo_hasdbxref_gard.sssom.tsv))
+, `new_proxy`, and `unmapped_proxy_or_direct` (which is same as the `gard_unmapped_terms` sheet). 
+2. `gard_unmapped_terms`: A flat list of GARD IDs that do not have existing mappings in Mondo, and no new proxy mappings found.
+3. `GARD source data`: Copy of source CSV provided by GARD.
+4. `gard_terms_mapping_status`: This output was mainly for us to provide to the GARD folks. This allows us to see at a 
+glance the GARD IDs that exist in the GARD data source as well as in our current Mondo mappings.
+5. `obsoleted_gard_terms_in_mondo`: This output was initially for us to provide to the GARD folks.This is a flat list of
+ GARD IDs that have xrefs in Mondo but are no longer in GARD.
+
+#### Assumptions
+**Mapping assumptions**  
+Joe and Nico interpreted the [GARD source data](https://docs.google.com/spreadsheets/d/1w5Xnzr5uNFcPrQqCT8mGBFHnGhwXDfQVzHKofw6kB7c/edit#gid=885755208) 
+and tried to determine the best predicates to use for mappings. The results are in the [`gard.sssom+` sheet](https://docs.google.com/spreadsheets/d/1w5Xnzr5uNFcPrQqCT8mGBFHnGhwXDfQVzHKofw6kB7c/edit#gid=1504531712). 
+This only applies to those rows where `gard_mondo_mapping_type`==`proxy_new`.
+1. **GARD::Orphanet**: `skos:exactMatch`: Where `DataSource`==`Orphanet`, value from the `SourceID` column is an Orphanet ID.
+2. **GARD::Orphanet**: `skos:broadMatch`: There is a `ParentOrphaCode` column. Any IDs from that column are considered broad matches.
+3. **GARD::OMIM**: `skos:exactMatch`: I’m unsure why they called it "`Orphanet+OMIM`", but based on what Yanji said, where 
+`DataSource`==`Orphanet+OMIM`, the value in the `SourceID` column is an OMIM ID.
+4. **GARD::OMIM**: `skos:narrowMatch`: These come from the `OmimMember` column. Rows has 0 to many MIM#s in that column. 
+Exception 1: when `DataSource`==`Orphanet+OMIM`. In that case, there is only 1 item in `OmimMember`, and it is equal to what 
+is in the `SourceID` column and considered an exact match. Exception 2: when `DataSource`==`GARD`, these are considered 
+related matches. Yanji told Joe that these rows are in development, so perhaps they might be exact, narrow, or perhaps 
+even broad.
+5. **GARD::OMIM**: `skos:relatedMatch`: For rows where `DataSource`==`GARD` (n=10), some of these rows have values in the 
+`OmimMember` column (n=5). In all these cases, it’s just 1 MIM#. Yanji mentioned in email that these rows were 
+tentative / being worked on. So I felt that without asking or taking a closer look, I was unsure if these would be 
+narrow or exact matches.
```

Comment:
Please add note on how you interpreted the subclassOf link

Reply:
Done!